src/node_report.cc

```diff
@@ -752,14 +853,15 @@ static void PrintStackFrame(FILE* fp, Isolate* isolate, Local<StackFrame> frame,
  * Function to print a native stack backtrace
  *
  ******************************************************************************/
-void PrintNativeStack(FILE* fp) {
+void PrintNativeStack(std::ostream& out) {
   void* frames[64];
-  fprintf(fp, "\n================================================================================");
-  fprintf(fp, "\n==== Native Stack Trace ========================================================\n\n");
+  out << "\n================================================================================";
+  out << "\n==== Native Stack Trace ========================================================\n\n";
 
   HANDLE hProcess = GetCurrentProcess();
   SymSetOptions(SYMOPT_LOAD_LINES | SYMOPT_UNDNAME | SYMOPT_DEFERRED_LOADS);
   SymInitialize(hProcess, nullptr, TRUE);
+  char buf[64];
```

Comment:
I wonder if 64 chars will always be enough for symbol+filename etc. Maybe we can afford to be more generous. The snprintf() will just truncate, so it's safe anyway.